internal/services/compute/windows_virtual_machine_scale_set_resource.go

```diff
@@ -81,7 +81,8 @@ func resourceWindowsVirtualMachineScaleSet() *pluginsdk.Resource {
 				return false
 			}),
 
-			pluginsdk.CustomizeDiffShim(func(ctx context.Context, diff *pluginsdk.ResourceDiff, v interface{}) error {
+			// Validate network interface auxiliary mode and sku requirements
+			pluginsdk.CustomizeDiffShim(func(ctx context.Context, diff *schema.ResourceDiff, v interface{}) error {
```

Comment:
I'm not sure this change and comment are needed here?
```suggestion
			pluginsdk.CustomizeDiffShim(func(ctx context.Context, diff *pluginsdk.ResourceDiff, v interface{}) error {```

Reply:
Fixed.